packages/core/utils/src/urlRelative.js

```diff
@@ -0,0 +1,7 @@
+// @flow strict-local
+
+import path from 'path';
+
+export default function urlRelative(from: string, to: string): string {
```

Comment:
Parcel 1 had `urlJoin` for this, which handled a few more edge cases (e.g. query strings and hashes).

Reply:
This isn't exactly the same, no? Maybe `urlJoin(path.dirname(from), to)` would be, but I'll have to check with hashes and querystrings.